Server/VikingVault.Services/UserService.cs

```diff
@@ -63,5 +64,24 @@ private BankAccount CreateBankAccount(User user, String currencyType)
                 Balance = 0.0f
             };
         }
+
+        public void DeleteUser(UserEmail userEmail)
```

Comment:
Why does this file has the same changes as the `int2019 28 admin dashboard add money` pull request, or the other way around?